README.md

```diff
@@ -8,23 +8,17 @@ CloudFront Adress
 
 ## Tasks
 
-- 1 - Task 4.1 is implemented - done (see add-data-scripts.sql)
-- 2 - TASK 4.2 is implemented lambda links are provided and returns data - done
-- 3 - TASK 4.3 is implemented lambda links are provided and products is stored in DB (call TASK 4.2 to see the product)
-- 4 - Your own Frontend application is integrated with product service (/products API) and products from product-service are represented on Frontend. Link to a working Front-End application is provided for cross-check reviewer.
+Reviewers should verify the lambda functions by invoking them through provided URLs.
```

Comment:
Please provide a link to your FE application at Pull request description

Reply:
As I mentioned above, see reedme file, please
https://github.com/Max-Korsakov/nodejs-aws-be/blob/task-5/README.md

Reply:
![image](https://user-images.githubusercontent.com/48759392/99491829-87bea080-297d-11eb-9e74-2344b9b2464a.png)